src/views/AccountView.vue

```diff
@@ -54,4 +75,24 @@
 
     await auth.delete();
   }
+
+  async function edit(index: number) {
+    if (settings[index].value === 'Ändern') {
+      settings[index].value = 'Speichern';
+      disabledInputs[index].value = false;
+    } else {
+      settings[index].value = 'Ändern';
+      disabledInputs[index].value = true;
+      const result = await auth.changeAccountData();
```

Comment:
Und den Anfang der Methode dann auch entsprechend anpassen. Z.B. so in die Richtung vllt?

```
disabledInputs[index].value = !disabledInputs[index].value;
if (disabledInputs[index].value) {
   // Änderungen speichern
}
```